infra/grafana/dashboards/service.json

```diff
@@ -0,0 +1,471 @@
+          "intervalFactor": 1,
+          "legendFormat": "periapsis",
+          "refId": "B",
+          "range": true
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Orbital Parameters",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        },
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        }
+      ]
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "Prometheus",
+      "fill": 1,
+      "gridPos": {
+        "h": 10,
+        "w": 30,
+        "x": 0,
+        "y": 10
+      },
+      "id": 5,
+      "legend": {
+        "alignAsTable": false,
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "rightSide": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "percentage": false,
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "repeat": null,
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "control_mode{} > 0",
+          "format": "table",
+          "interval": "1s",
+          "instant": true,
+          "intervalFactor": 1,
+          "legendFormat": "control_mode",
+          "refId": "A",
+          "range": true
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Control Mode",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "transformations": [
+        {
+          "id": "convertFieldType",
+          "options": {
+            "conversions": [
+              {
+                "destinationType": "enum",
+                "enumConfig": {
+                  "text": [
+                    "PAD",
+                    "ROLL",
+                    "PITCH_OVER",
+                    "MAX_Q",
+                    "THROTTLE_BUCKET_EXIT",
+                    "MECO_PREP",
+                    "MECO",
+                    "S2_INIT",
+                    "S2_LOOP1",
+                    "S2_LOOP2",
+                    "S2_LOOP3",
+                    "COAST",
+                    "ABORT"
```

Comment:
it's annoying all these enums are duplicated between the dashboard definition and the python code.

probably best long term solution would be to generate this list based on python enum state at code build time.

in meantime remember to add control mode states both here and in the launch_utils control mode enum for proper display